### PR TITLE
docs(marketing): add Zenika trainings in resources.json

### PR DESCRIPTION
**Pull request by EmmanuelDemey**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Add on-site trainings made by Zenika (France)

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -782,6 +782,12 @@
             "title": "SFEIR School (French)",
             "url": "https://school.sfeir.com/project/sa200/"
           },
+          "zenika-angular": {
+            "desc": "Angular trainings delivered by Zenika (FRANCE)",
+            "rev": true,
+            "title": "Angular Trainings (French)",
+            "url": "https://training.zenika.com/fr/training/angular2/description"
```
Comment (gkalpak) [inline aio/content/marketing/resources.json]: `angular2` 😱 😱 😱

Comment (Gillespie59) [reply aio/content/marketing/resources.json]: Yes only in the URL. I only have access to the content of the page.

**Comment by Gillespie59** [comment]
Done @brandonroberts

**Review by StephenFluin — APPROVED** [review]
We'd like the URL to be updated in the future, but this looks OK for now.

**Pull request closed without merging**